mix.lock

```diff
@@ -1,2 +1,2 @@
-%{"erlzk": {:git, "git://github.com/huaban/erlzk.git", "a005aab956e88686d8d9e719cb3937ae91c98b68", [ref: "a005aab956e88686d8d9e719cb3937ae91c98b68"]},
-  "uuid": {:package, "0.1.5"}}
+%{"erlzk": {:git, "https://github.com/huaban/erlzk.git", "a005aab956e88686d8d9e719cb3937ae91c98b68", [ref: "a005aab956e88686d8d9e719cb3937ae91c98b68"]},
```

Comment:
this change was introduced automatically after `mix deps.get` with mix version 1.1.1